post-script.rhai

```diff
@@ -1,4 +1,4 @@
-if variable::get("mcu") != "esp32h2" && variable::get("wifi"){
+if variable::get("wifi"){
     print("\nFor more information and examples of esp-wifi showcasing Wifi,BLE and ESP-NOW, see https://github.com/esp-rs/esp-wifi/blob/main/examples.md\n");
```

Comment:
Just noticed that the URL is now invalid since we change the repo structure, can we update it to https://github.com/esp-rs/esp-wifi/blob/main/esp-wifi/docs/examples.md or https://github.com/esp-rs/esp-wifi/tree/main/esp-wifi/examples

Reply:
Oh, totally missed that ... probably we should even include the esp-wifi version: https://github.com/esp-rs/esp-wifi/blob/v0.2.0/esp-wifi/docs/examples.md so users will see the code that matches their version (since `main` is basically next version) 🤔

Reply:
Is there any reason to have the `examples.md` under `docs` folder and not in `esp-wifi/examples/readme.md`?

Reply:
probably no other reason than often repos have additional documentation in a `docs` folder